backend/blueprint/api/utils/response.py

```diff
@@ -1,4 +1,5 @@
-from api.config import urlConfig
+from instance import urlConfig
+
```

Comment:
No need of this as the previous space was fine according to hound configurations :+1: